cpp/src_prims/label/classlabels.cuh

```diff
@@ -39,43 +39,35 @@ using namespace MLCommon;
  * \tparam math_t numeric type of the arrays with class labels
  * \param [in] y device array of labels, size [n]
  * \param [in] n number of labels
- * \param [out] y_unique device array of unique labels, unallocated on entry,
+ * \param [out] unique device array of unique labels, unallocated on entry,
```

Comment:
Thanks for noticing, I'll update the documentation. There's two `getUniqueLabels` functions, one in RAFT and an other one in cuML. I modified the one in RAFT for it to take a `rmm::device_uvector` in. It will be used everywhere in the cuML code. Then, there's a cuML version of it (and its test), that is exposed with raw pointers. It could be left there to be part of the C API or otherwise deleted.